parachain/runtime/kintsugi/src/lib.rs

```diff
@@ -745,6 +743,11 @@ pub type LocalAssetTransactor = MultiCurrencyAdapter<
 
 pub use currency_id_convert::CurrencyIdConvert;
 
+impl orml_xcm::Config for Runtime {
+    type Event = Event;
+    type SovereignOrigin = EnsureRootOrAllTechnicalCommittee;
```

Comment:
@gregdhill is this correct? We need governance/sudo to be able to make these calls

Reply:
You'll need to use `EnsureRoot` for that.